experimenter/Dockerfile

```
@@ -111,14 +92,13 @@ COPY --from=node-builder /experimenter/experimenter/theme/static_src/node_module
 COPY --from=node-builder /experimenter/experimenter/nimbus-ui/node_modules/ /experimenter/experimenter/nimbus-ui/node_modules/
 COPY --from=node-builder /experimenter/node_modules/ /experimenter/node_modules/
 
-# Rust packages
-COPY --from=rust-builder /application-services/megazord/fml.py /application-services/megazord/rust_fml.py
-COPY --from=rust-builder /application-services/megazord/nimbus.py /application-services/megazord/rust_sdk.py
-COPY --from=rust-builder /application-services/megazord/errorsupport.py /application-services/megazord/rust_errorsupport.py
-COPY --from=rust-builder /application-services/megazord/remote_settings.py /application-services/megazord/rust_remote_settings.py
-# The uniffi.toml file setting cdylib_name is set to "cirrus", which means the generated python is expecting
-# the library to be called libcirrus.so.
-COPY --from=rust-builder /application-services/target/release/libnimbus_experimenter.so /application-services/megazord/libcirrus.so
+# Copy nimbus-experimenter megazord
+COPY --from=as-builder /application-services/nimbus-experimenter/fml.py /application-services/megazord/rust_fml.py
+COPY --from=as-builder /application-services/nimbus-experimenter/fml.py /application-services/megazord/rust_fml.py
+COPY --from=as-builder /application-services/nimbus-experimenter/nimbus.py /application-services/megazord/rust_sdk.py
+COPY --from=as-builder /application-services/nimbus-experimenter/errorsupport.py /application-services/megazord/rust_errorsupport.py
+COPY --from=as-builder /application-services/nimbus-experimenter/remote_settings.py /application-services/megazord/rust_remote_settings.py
+COPY --from=as-builder /application-services/nimbus-experimenter/lib/libnimbus_experimenter.so /application-services/megazord/libnimbus_experimenter.so
```

Again, let's do all the preparation of the megazord in the `fetch-application-services.sh` script, so we're just left with

```suggestion
COPY --from=as-builder /application-services/nimbus-experimenter /application-services/megazord
```